### ISSUE TITLE
 libssl.so.1.0.0 issue

### ISSUE THREAD

**Issue by zer09** [nosqlclient/nosqlclient#434]
I am not able to run nosqlclient on fedora 27.

## Expected Behavior
It should run without problems.

## Current Behavior
IT failed to run with this following error
```
error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory

[NOSQLCLIENT] [MONGOD-EXIT] 127

```

## Steps to Reproduce (for bugs)
1. download the portable version [Linux X64](https://github.com/nosqlclient/nosqlclient/releases/download/2.2.0/linux-portable-x64.zip)
2. Extract the zip
3. execute the ```Nosqlclient```

## Your Environment
OS: Fedora 27
Nosqlclient version: 2.2.0

**Comment by rsercano** [comment]
Hello @zer09 

Nosqlclient internally uses a **mongo binary** for itself, therefore if you may change this binary with the fedora one, it should work. To change internal mongo binary you may use [MONGO_URL ](https://www.nosqlclient.com/docs/start.html) environment variable. 

Let me know if it works for you.

**Comment by zer09** [comment]
I can't remember I changed something on the mongo, it was just a default install.
But i tried this in terminal.

```
export MONGO_URL=mongodb://localhost:27017
./Nosqlclient
```
Still got the same error.

**Comment by rsercano** [comment]
You need an external mongo instance to make nosqlclient work. Therefore did you start another mongo instance before starting nosqlclient ? And also try to put a database name into MONGO_URL environment variable

**Comment by zer09** [comment]
Yes, I already have mongod instance running. also tried to add the database name still got same error.

I took a screenshot of it.

![screenshot from 2018-05-12 15-53-21](https://user-images.githubusercontent.com/780780/39955206-95ba1cec-55fd-11e8-9a41-c86445c98ebd.png)

**Comment by rsercano** [comment]
So weird, it shouldn't try to start it's own internal mongod instance since MONGO_URL environment variable is set, can you try to put your own `mongod` instance to `/mnt/Busy Driver/tmp/Nosqlclient-linux-x64/resources/app/bin` and let me know if it works for you ?

**Comment by rsercano** [comment]
will be resolved with the duplicate #384
